### PR TITLE
Import the appropriate yaml

### PR DESCRIPTION
**Pull request by dieris**
I think this should complement  https://github.com/BBN-Q/Auspex/commit/0c1d897a4df5d7ad3955ba82497d2d0edcb1cdb0

### CODE REVIEW

File: src/auspex/filters/io.py [hunk 1]
```diff
@@ -19,7 +19,10 @@
 import re
 import pandas as pd
 from shutil import copyfile
-from ruamel.yaml import YAML
+try:
+    from ruamel.yaml import YAML
+except:
+    from ruamel_yaml import yaml_object
```
Comment (grahamrow) [inline src/auspex/filters/io.py]: should this be `as YAML` or something?

Comment (grahamrow) [reply src/auspex/filters/io.py]: In `config.py` we have, for example:
```python
try:
    import ruamel.yaml as yaml
except:
    import ruamel_yaml as yaml
```

Comment (dieris) [reply src/auspex/filters/io.py]: I think you're right. I got confused with all these ~namesakes

Comment (dieris) [reply src/auspex/filters/io.py]: actually, I don't think `YAML` is used either...

**Pull request merged**